vaadin-spring-tests/pom.xml

```diff
@@ -153,6 +153,7 @@
 
         <module>test-spring-security-flow</module>
         <module>test-spring-security-fusion</module>
+        <module>test-spring-security-fusion-custom-context</module>
```

Comment:
The ohter tests with a context path are named `test-spring-boot-contextpath` and `test-spring-security-flow-contextpath` so this should be `test-spring-security-fusion-contextpath`